bugbug/bugzilla.py

```diff
@@ -344,6 +344,31 @@ def get_product_component_count(months: int = 12) -> dict[str, int]:
     return bugs_number
 
 
+def get_active_product_components(products=[]) -> set[tuple[str, str]]:
+    r = utils.get_session("bugzilla").get(
+        "https://bugzilla.mozilla.org/rest/product",
+        params={
+            "type": "accessible",
+            "include_fields": [
+                "name",
+                "is_active",
+                "components.name",
+                "components.is_active",
+            ],
+            "names": products,
+        },
+        headers={"X-Bugzilla-API-Key": Bugzilla.TOKEN, "User-Agent": "bugbug"},
+    )
+    r.raise_for_status()
+
+    return set(
+        (product["name"], component["name"])
+        for product in r.json()["products"]
+        for component in product["components"]
+        if product["is_active"] and component["is_active"]
+    )
```

Comment:
nit: checking if the product is active before iterating over the components improves the readability and the performance: 
```suggestion
    return {
        (product["name"], component["name"])
        for product in r.json()["products"]
        if product["is_active"]
        for component in product["components"]
        if component["is_active"]
    }
```

Reply:
Done in b9ea52cd87e0ebe2312d604cd5e84fca42f3d2b2.